src/main/java/com/prgrms/modi/party/domain/Party.java

```diff
@@ -255,4 +256,13 @@ public Party build() {
 
     }
 
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
```

Comment:
```suggestion
        return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE)
```

![스크린샷, 2021-12-08 16-49-09](https://user-images.githubusercontent.com/58318786/145169343-7b53c037-32fe-4699-9a57-a11aaf44055b.png)

![스크린샷, 2021-12-08 16-49-19](https://user-images.githubusercontent.com/58318786/145169350-31271117-ae62-4d52-b28c-fdc2c629c752.png)

* 저도 까먹고 안 넣은 부분 있는데ㅎㅎ 이런 차이라고 합니다!